src/components/App/App.jsx

```diff
@@ -11,6 +12,7 @@ function App() {
 
   const [durationLong, setDurationLong] = useState(false)
```

Comment:
why did you named it duration long? Why not just duration? Or is there also a duration short?

Reply:
There is a short duration which is only applied, if there the state of the long duration is not active. The PR doesn't show the full code unfortunately.